### PR TITLE
Update readme.md to clarify code type

### PR DESCRIPTION
**Pull request by himapatel2011**
Modified readme.md to improve specification on code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ to your API token file:
 export VOXEL51_API_TOKEN="/path/to/your/api-token.json"
 ```
 
-Alternatively, you can permanently activate a token with:
+Alternatively, you can permanently activate a token with following js code. Run this code in Node js:
```
Comment (hoddr) [inline README.md]: Say something like: "... can permanently activate a token with the following JavaScript code. Open the `nodejs` REPL and run: ... "

File: README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ After you have activated an API token, you have full access to the API.
 
 ## Example Usage
 
-To initialize an API session, issue the following commands:
+To initialize an API session, issue the following all  commands in Node js:
```
Comment (hoddr) [inline README.md]: "... issue the following ~~all~~ commands in `nodejs`:

File: README.md [hunk 3]
```diff
@@ -70,13 +70,13 @@ function pprint(obj) {
 
 ### Analytics
 
-List available analytics:
+List all available analytics:
```
Comment (hoddr) [inline README.md]: I don't think the `all` is necessary. Perhaps `List the available analytics:`. "all" is implied here

File: README.md [hunk 4]
```diff
@@ -70,13 +70,13 @@ function pprint(obj) {
...
 
 ```js
 let analytics = api.listAnalytics();
 ```
 
-Get documentation for an analytic:
+Get documentation for an analytic. Asign certain `analyticId` as argument:
```
Comment (hoddr) [inline README.md]: Typo on `asign`. Should be `assign`. Better version:

"Get documentation for an analytic, where the `analyticId` variable matches the analytic's ID".

File: README.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ api.getAnalyticDoc(analyticId).then(function(doc) {
 
 ### Data
 
-Upload data to the cloud:
+Upload data to the cloud. pass path to the file along with file name as an argument:
```
Comment (hoddr) [inline README.md]: "Upload data to cloud storage, where `uploadDataPath` is the absolute or relative path to the local date file."

File: README.md [hunk 6]
```diff
@@ -98,7 +98,7 @@ api.uploadData(uploadDataPath).then(function(metadata) {
 });
 ```
 
-List uploaded data:
+List all uploaded data:
```
Comment (hoddr) [inline README.md]: Again, I find the `all` to be redundant.

File: README.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ api.listJobs().then(function(metadata) {
 });
 ```
 
-Create a job request:
+Create a job request to run on a data. Pass certain `Analytic-Name(Type)` as `<analytic>`, particular `Data Id` as `dataId`, give data type `video` or `images` as `<input>`,and certain paramter and it's value based on analytic type you choose to run. For more information on `input` and `parameters` refers to `analytic.json`   :
```
Comment (hoddr) [inline README.md]: "Create a job request to perform an analytic on a data, where `<analytic>` is the analytic's name, `dataId` is the desired data file's unique ID, and any `<param#>` values are any of the permitted parameters based on the analytic's documentation."

**Review by hoddr — CHANGES_REQUESTED** [review]
See comments for some minor fixes.

File: README.md (hunk 3 above)
Comment (himapatel2011) [reply README.md]: OK. I will change it back to normal.

File: README.md (hunk 2 above)
Comment (himapatel2011) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (himapatel2011) [reply README.md]: ok

File: README.md (hunk 7 above)
Comment (himapatel2011) [reply README.md]: ok. thanks. I will modify the text

File: README.md (hunk 1 above)
Comment (himapatel2011) [reply README.md]: right. ok

File: README.md (hunk 6 above)
Comment (himapatel2011) [reply README.md]: yeah, that's true.

**Comment by himapatel2011** [comment]
Review suggestions fixed. Changes are available here, https://github.com/voxel51/api-js/pull/5/commits/6c4b641c06018cb49731d87df002371de002e836

**Review by hoddr — APPROVED** [review]
LGTM - awaiting BM approval for closure

**Pull request merged**